app/DetailPage/CardPrice/Page.jsx

```diff
@@ -0,0 +1,74 @@
+"use client"
+import { Box, Button, Card, CardActions, CardContent, CardMedia, Typography } from '@mui/material'
+import React, { useState } from 'react'
+import { GetDataById } from '../[Detail]/util'
+import { useRouter } from 'next/navigation'
+
+export default function CardPrice({ passParam }) {
+
+    const router = useRouter()
+    // usestate place
+    const [hoverImg, setHoverImg] = useState(false)
+
+    let data = GetDataById(passParam)
+
+    const handleGetByID = (e) => {
+        const IdProduct = data.id
+        router.push(`/PaymentPage/${IdProduct}`)
+    }
+
+
+    return (
+        <Card
+            sx={
+                {
+                    position: "fixed",
+                    top: "170px",
+                    right: "65px",
+                    maxWidth: "350px",
+                    boxShadow: "10",
+                    userSelect: "none",
+                    paddingBottom: "34px",
+                    borderRadius: "10px",
+                }
+            }
+        >
+            <Box
+                sx={
+                    {
+                        overflow: "hidden",
+                        width: "100%",
+                        height: "200px",
+                    }
+                }
+                onMouseEnter={() => setHoverImg(true)}
+                onMouseLeave={() => setHoverImg(false)}
+            >
+                <CardMedia
+                    component="img"
+                    alt="green iguana"
+                    height="200"
+                    image="https://images.unsplash.com/photo-1696958363946-b58f40cda426"
+                    sx={
+                        {
+                            transition: "all 0.2s",
+                            transform: hoverImg ? "scale(1.2)" : "scale(1)"
+                        }
+                    }
+                />
+            </Box>
+            <CardContent>
+                <Typography gutterBottom variant="h5" component="div" sx={{ fontWeight: "700", color: "#7F56D9" }}>
```

Comment:
Can you change all these into a format of a bank card? It would be great if you do that

Reply:
I dont get it ?